components/suggest/src/testing/mod.rs

```diff
@@ -33,6 +40,19 @@ impl JsonExt for JsonValue {
         self_map.append(other_map);
         self
     }
+
+    fn remove<Q>(mut self, key: &Q) -> JsonValue
+    where
+        String: Borrow<Q>,
+        Q: ?Sized + Ord + Eq + Hash,
+    {
+        let self_map = match &mut self {
```

Comment:
Just an FYI, there is another handy way to do "match this pattern and bind the matched, otherwise panic".

```rs
    let JsonValue::Object(obj) = &mut self else {
        panic!("remove called on non-object: {self:?}");
    };
    obj.remove(key);
```

Reply:
Thanks, I updated `JsonExt::merge` too.